modules/imaging_browser/php/imagingbrowserrowprovisioner.class.inc

```diff
@@ -166,7 +166,7 @@ class ImagingBrowserRowProvisioner extends \LORIS\Data\Provisioners\DBRowProvisi
         // $scan_types are set in the configuration module
         $modalities_subquery = '';
         foreach ($case_desc as $key => $value) {
-            $modalities_subquery = "$value as $scan_id_types[$key]_QC_Status";
+            $modalities_subquery .= "$value as $scan_id_types[$key]_QC_Status,";
```

Comment:
```suggestion
            $modalities_subquery .= "$value as $scan_id_types[$key]_QC_Status, ";
```
not sure is this is where the linter removed the space, but adding one here will keep the clauses cleanly distinct (if ever debugging)

Reply:
Good point! I've committed the suggestion.